localization/ndt_scan_matcher/test/test_fixture.hpp

```diff
@@ -0,0 +1,89 @@
+// Copyright 2023 Autoware Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef TEST_FIXTURE_HPP_
+#define TEST_FIXTURE_HPP_
+
+#include "../include/ndt_scan_matcher/ndt_scan_matcher_core.hpp"
+#include "stub_initialpose_client.hpp"
+#include "stub_pcd_loader.hpp"
+#include "stub_sensor_pcd_publisher.hpp"
+#include "stub_trigger_node_client.hpp"
+
+#include <ament_index_cpp/get_package_share_directory.hpp>
+#include <rclcpp/rclcpp.hpp>
+
+#include <gtest/gtest.h>
+#include <rcl_yaml_param_parser/parser.h>
+
+#include <memory>
+#include <string>
+#include <vector>
+
+class TestNDTScanMatcher : public ::testing::Test
+{
+protected:
+  void SetUp() override
+  {
+    const std::string yaml_path = ament_index_cpp::get_package_share_directory("ndt_scan_matcher") +
+                                  "/config/ndt_scan_matcher.param.yaml";
+
+    rcl_params_t * params_st = rcl_yaml_node_struct_init(rcl_get_default_allocator());
+    if (!rcl_parse_yaml_file(yaml_path.c_str(), params_st)) {
+      std::cout << "Failed to parse yaml file" << std::endl;
+      return;
+    }
+
+    const rclcpp::ParameterMap param_map = rclcpp::parameter_map_from(params_st, "");
+    rclcpp::NodeOptions node_options;
+    for (const auto & param_pair : param_map) {
+      for (const auto & param : param_pair.second) {
+        node_options.parameter_overrides().push_back(param);
+      }
+    }
+    node_ = std::make_shared<NDTScanMatcher>(node_options);
+    rcl_yaml_node_struct_fini(params_st);
+
+    // prepare tf_static "base_link -> sensor_frame"
+    tf_broadcaster_ = std::make_shared<tf2_ros::TransformBroadcaster>(node_);
```

Comment:
```suggestion
    tf_broadcaster_ = std::make_shared<tf2_ros::StaticTransformBroadcaster>(node_);
```
https://docs.ros.org/en/humble/Tutorials/Intermediate/Tf2/Writing-A-Tf2-Static-Broadcaster-Cpp.html#

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/6649/commits/d538dcfff442328baefa53c6fda61f1c5c925c84